src/com/company/Constants.java

```diff
@@ -0,0 +1,9 @@
+package com.company;
+
+public class Constants {
+    final static String FILE_TYPE_TXT = ".txt";
+    final static String FILE_TYPE_JSON = ".json";
+    final static String SPECIAL_CHAR_REMOVAL_REGEX = "[^a-zA-Z0-9@-]";
+    final static String SINGLE_SPACE = " ";
+    static String STATUS_FAILURE = "Failure";
```

Comment:
Did address these
1. Should be final
2. You should have another constant for success with proper value STATUS_SUCCESS and use that

Reply:
done with this changes